homeworks/11-animations/index.css

```diff
@@ -0,0 +1,98 @@
+* {
+  padding: 0;
+  margin: 0;
+  --delay: 180ms;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  background-color: #f0f0f0;
+}
+
+#middleCircleArc {
+  animation: middleCircleArcMovement calc(var(--delay) * 5) cubic-bezier(.17,.67,.83,.67) infinite;
```

Comment:
In CSS common code style practice is to add space after comma for readability `cubic-bezier(.17, .67, .83, .67)`